.env.example

```
@@ -1,3 +1,3 @@
 MONGODB_URI = <MONGODB_URI> ( put a mongodb url here)
-BASE_URL = https://harshgoel.me   (this is my api URL, change it to http://localhost:3000, in case you tend to use your database)
+BASE_URL = http://localhost:3000  (this is my api URL, change it to http://localhost:3000, in case you tend to use your database)
```

Please revert it back
```suggestion
BASE_URL = https://harshgoel.me  (this is my api URL, change it to http://localhost:3000, in case you tend to use your database)
```